src/lib/components/utils/acm-toaster.svelte

```diff
@@ -58,7 +57,7 @@
     align-self: center;
   }
 
-  @media (min-width: 800px) {
+  @media (min-width: 900px) {
```

Comment:
can you also fix #408 with this

Reply:
Toasts centered in landscape during my testing. will double check tho

Reply:
referring to the fact that all the toasts take the width of the widest existing toast so it snaps back when it disappears